fenix/app/src/main/java/org/mozilla/fenix/components/FenixSnackbar.kt

```diff
@@ -161,7 +161,7 @@ class FenixSnackbar private constructor(
                         // can't intelligently position the snackbar on the upper most view.
                         // Ideally we should not pass ContentFrameLayout in, but it's the only
                         // way to display snackbars through a fragment transition.
-                        (view is ContentFrameLayout || !dynamicToolbarEnabled)
+                        (view is ContentFrameLayout || view is CoordinatorLayout || !dynamicToolbarEnabled)
```

Comment:
Sorry drive-by: Just curious if we also tested the other snackbar for other features (collections, history deletion, synced tabs syncing, etc) to make sure they are still behaving accordingly?

Reply:
@gabrielluong I actually had not, thanks for prompting me, having a look into this